.github/workflows/node-ci.yml

```diff
@@ -86,55 +91,55 @@ jobs:
 
             - name: Check formatting
               run: |
-                  yarn -s prettier --check .
+                  pnpm -s prettier --check .
```

Comment:
I don't think `-s` has any effect for `pnpm`. At least, it didn't seem to make any difference to the output for this command. `pnpm` is non-noisy by default.